### ISSUE TITLE
Judging of valid submissions without POC

### ISSUE THREAD

**Issue by thereksfour** [code-423n4/org#74]
Valid submissions without POC should not be nullified.
Considering the short duration of the C4 contest, warden may prefer to spend time on new findings rather than writing POC.
Currently C4 already has a 30% bonus for high quality submissions, which encourages warden to write POC.
But if we further slash valid submissions without POC, then warden will have to spend time "optimizing" their submissions instead of finding new findings.

**Comment by IllIllI000** [comment]
Since the findings aren't public yet, linking to the backstage discussion for the two examples that triggered this post: https://discord.com/channels/810916927919620096/976603323450941440/1048076707493580821

I agree that a POC should not be required, but the submission still should state what is wrong, what the impact is/how user funds are lost. With 287, scanning through the submission, I can't tell what the problem is, or how severe it is

**Comment by csanuragjain** [comment]
I am also of opinion that POC must not be a hardcore requirement but definitely should contain step by step poc showing what exactly is wrong and should be something which is easy for judges to understand

**Comment by thereksfour** [comment]
> Since the findings aren't public yet, linking to the backstage discussion for the two examples that triggered this post: https://discord.com/channels/810916927919620096/976603323450941440/1048076707493580821
> 
> I agree that a coded POC should not be required, but the submission still should state what is wrong, what the impact is/how user funds are lost. With 287, scanning through the submission, I can't tell what the problem is, or how severe it is

My mistake, after a closer look, #287 should be a duplicate of #229

**Comment by dmvt** [comment]
I think this is already pretty well outlined in the [Update to award formula, scoring, and contest rules](https://github.com/code-423n4/org/discussions/50) discussion. 

![Screenshot 2022-12-02 at 22 51 14](https://user-images.githubusercontent.com/1116695/205402898-d04bd90f-b5b6-4274-96e9-0d686f702a8b.png)

The issues which triggered this discussion were both labeled as high risk and therefore require, at a minimum, a walkthrough of how the exploit works. 

> But if we further slash valid submissions without POC, then warden will have to spend time "optimizing" their submissions instead of making more findings.

Please keep in mind that as judges, we have a maximum of 7 days to dedup the findings. Everyone is working under reasonable timelines. IMO, the ability to do so starts with the wardens being clear about their submissions.

**Comment by thereksfour** [comment]
@dmvt 
I agree with you, but I think the nullification or unsatisfactory label is a bit too strict.
For a valid submission, I think its value comes first from its ability to identify the root cause of the vulnerability, and then a clear proof. The ratio might be 50/50
`Not higher than` does not mean 0
```
Submissions which judges deem insufficiently proven will not be eligible for anything higher than a satisfactory score.
```
When a vulnerability does exist, but warden doesn't provide sufficient proof, I think the `partial credit` label is enough

**Comment by dmvt** [comment]
This conversation continued in backstage, so dropping it here for future readers / participants:

https://discord.com/channels/810916927919620096/976603323450941440/1048406475556536351
<img width="909" alt="Screenshot 2022-12-03 at 11 08 51" src="https://user-images.githubusercontent.com/1116695/205437929-49d1d36e-9e0c-4c90-8edb-b6fafee89fbf.png">
<img width="907" alt="Screenshot 2022-12-03 at 11 10 34" src="https://user-images.githubusercontent.com/1116695/205438013-5caa0a4d-6eb2-403a-af91-c9dda8406681.png">
